docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst

```diff
@@ -13,8 +13,111 @@ Case definition in GBD 2017
 Neonatal sepsis and other neonatal infections are infections during the neonatal
 period that advance to a systemic bloodstream infection, the underlying cause of
 which can be meningitis, gastroenteritis, or other aetiologies. Neonatal
-pneumonia, however, is not included -- it is captured in our modelling of
+pneumonia, however, is not included -- it is captured in the modelling of
 pneumonia as a separate entity.
 
 .. todo::
-   Describe cause model
+   Add more information and references. In particular, find data about differences in global and 
+   gbd definitions, global prevalence and relation to folic acid during pregnancy.
+
+Modeling Measles in GBD 2017
```

Comment:
Should be NN Sepsis instead of Measles. Or, if we follow James's suggestion for the cause model template, to avoid copy/past errors like this, we could omit the cause name entirely and just call this section something like _GBD 2017 Modeling Strategy_.